content/en/docs/Getting Started/_index.md

```diff
@@ -373,11 +373,11 @@ tkn task start echo --dry-run > taskRun.yaml
 kubectl apply -f taskRun.yaml
```

Comment:
Hi 👋 @jjasghar.
Thank you so much for making this pr!
I tried running the following.
```yaml
apiVersion: tekton.dev/v1alpha1
kind: TaskRun
metadata:
  generatedName: echo-run-
spec:
  taskRef:
    name: echo
```
When I run:
```bash
kubectl apply -f taskRun.yaml
```
I get this error 
```bash
error: error when retrieving current configuration of:
Resource: "tekton.dev/v1beta1, Resource=taskruns", GroupVersionKind: "tekton.dev/v1beta1, Kind=TaskRun"
Name: "", Namespace: "tekton-pipelines"
from server for: "echo.yaml": resource name may not be empty
```
This seems to be a problem with kubectl. However, this problem goes away if you use:

```bash
kubectl create -f taskRun.yaml
```
This is because "When creating PipelineRuns/TaskRuns that use generateName with kubectl, kubectl create must be used as a new run is created each time. However, it is a common mistake to run kubectl apply and get the following error message". [link](https://github.com/tektoncd/pipeline/issues/2113)

Reply:
oh nvm, you made a second pr.